server/channels/store/sqlstore/channel_store.go

```diff
@@ -1231,7 +1256,7 @@ func (s SqlChannelStore) GetMoreChannels(teamId string, userId string, offset in
 	channels := model.ChannelList{}
 	err := s.GetReplica().Select(&channels, `
 		SELECT
-			Channels.*
+			`+strings.Join(channelSliceColumns("Channels"), ", ")+`
```

Comment:
Could you change this query to be built by the query builder? While this is probably safe as-is, concatenating this onto the query looks too susceptible to SQL injection for me

Reply:
done.